libzeth/tests/circuits/merkle_tree_test.cpp

```diff
@@ -4,50 +4,53 @@
 
 #include "libzeth/circuits/merkle_tree/merkle_path_authenticator.hpp"
 #include "libzeth/circuits/merkle_tree/merkle_path_selector.hpp"
-#include "libzeth/circuits/mimc/mimc_mp.hpp"
+#include "libzeth/circuits/mimc/mimc_mp_gadget.hpp"
 
 #include <gtest/gtest.h>
 #include <libff/algebra/curves/alt_bn128/alt_bn128_pp.hpp>
 
 using namespace libzeth;
 
+template<typename FieldT>
+using MiMCe7_permutation_gadget = MiMC_permutation_gadget<FieldT, 7, 91>;
+
 // Instantiation of the templates for the tests.  Data here assumes alt_bn128
 using ppT = libff::alt_bn128_pp;
-using FieldT = libff::Fr<ppT>;
-using HashTreeT = MiMC_mp_gadget<FieldT, MiMCe7_permutation_gadget<FieldT>>;
+using Field = libff::Fr<ppT>;
+using HashTreeT = MiMC_mp_gadget<Field, MiMCe7_permutation_gadget<Field>>;
```

Comment:
Shouldn't we also rename `ppT` -> `pp` and `HashTreeT` -> `HashTree` here?

Reply:
Renamed.